wear/src/main/java/chat/rocket/android/main/ui/MainNavigationAdapter.kt

```diff
@@ -0,0 +1,34 @@
+package chat.rocket.android.main.ui
+
+import android.content.Context
+import android.graphics.drawable.Drawable
+import androidx.wear.widget.drawer.WearableNavigationDrawerView
+import chat.rocket.android.R
+
+class MainNavigationAdapter(ctx: Context) :
+    WearableNavigationDrawerView.WearableNavigationDrawerAdapter() {
+
+    private val context: Context = ctx
+
+    override fun getItemText(pos: Int): CharSequence {
+        when (pos) {
+            0 -> return context.getString(R.string.title_chat_rooms)
+            1 -> return context.getString(R.string.title_settings)
+            2 -> return context.getString(R.string.title_logout)
+        }
+        return ""
+    }
+
+    override fun getItemDrawable(pos: Int): Drawable {
+        when (pos) {
```

Comment:
You can add `else` here: `else -> context.getDrawable(R.drawable.ic_chat_rooms_white_24dp)` and remove the `return`